Sources/Misc/Atomic.swift

```diff
@@ -73,3 +73,11 @@ internal final class Atomic<T> {
     }
 
 }
+
+extension Atomic: ExpressibleByNilLiteral where T: OptionalType {
```

Comment:
this may be somewhat redundant, but I think it'd be nice to add a comment like: 
```swift
// this is syntactic sugar that allows us to initialize an optional Atomic value to by directly assigning nil, 
// i.e.: let foo : Atomic<Foo?> = nil instead of the more indirect let foo : Atomic<Foo?> = .init(nil)
```

Reply:
Great idea, added!